cli/src/semgrep/commands/wrapper.py

```diff
@@ -5,9 +5,9 @@
 from typing import NoReturn
 from typing import Optional
 
+from semgrep import state
```

Comment:
I like that. I dunno why we don't do that more often, why people import all the symbols in a module
instead of simply qualifying those symbols.